src/test/groovy/com/github/gradle/node/yarn/task/YarnProxy_integTest.groovy

```diff
@@ -47,10 +46,7 @@ class YarnProxy_integTest extends AbstractIntegTest {
                     request.removeHeader("host")
                     def targetHost = "registry.npmjs.org"
                     request.withHeader("host", targetHost)
-                    if (secure) {
```

Comment:
wasn't really sure what the deal with this was? npm registry is always doing to be secure upstream, you get a redirect if you connect over port 80.

Reply:
Since Java supports both a HTTP and a HTTPS proxy we used to test that both worked, with HTTP support dropped I gave a brief try fixing the tests but got unpredictable results and instead disabled the HTTP tests (there's a separate issue for the proxy tests)